### PR TITLE
[PLE-650] Add TFLint validate tags rule and include AWS tag checks in ensure_default_tags

### PR DESCRIPTION
**Pull request by nicolajv**
- Add rule for allowing only a subset of team names
- Add docs

### CODE REVIEW

File: rules/ensure_default_tags_rule.go [hunk 1]
```diff
@@ -59,20 +79,120 @@ func (r *EnsureDefaultTagsRule) Check(runner tflint.Runner) error {
...
+
+func (r *AWSRunner) DecodeRuleConfig(ruleName string, ret interface{}) error {
+	v := reflect.ValueOf(ret).Elem()
+	v.Set(reflect.ValueOf(struct {
+		Tags    []string `hclext:"tags"`
+		Exclude []string `hclext:"exclude,optional"`
+	}{
+		Tags:    config.Tags,
+		Exclude: config.Exclude,
+	}))
```
Comment (vyrwu) [inline rules/ensure_default_tags_rule.go:195]: Won't this override the existing `ret` (which I assume is the value of the decoded config)?

File: rules/ensure_default_tags_rule_test.go [hunk 2]
```diff
@@ -5,73 +5,155 @@ import (
...
+			}
+			
+			resource "aws_instance" "ec2_instance" {
+				region = "eu-west-1"
+			}`,
+			Config: `
+			rule "ensure_default_tags" {
+			  enabled   = true
+			  tags		= ["team"]
+			}`,
```
Comment (vyrwu) [inline rules/ensure_default_tags_rule_test.go:37]: Would it make sense to create a test case with multiple config rules, and ensure they both still work together?

Comment (nicolajv) [reply rules/ensure_default_tags_rule_test.go:37]: I don't think so, these rules are practically completely independent except for residing in the same namespace. They will be running alongside of hundreds of default TFLint rules

**Review by vyrwu — APPROVED** [review]
Awesome work!!!

**Pull request merged**